src/components/MatrixBlock/MatrixBlock.module.scss

```diff
@@ -0,0 +1,29 @@
+@import "../../assets/styles/variables";
+@import "../../assets/styles/mixins";
+
+.matrixTable {
+	@include flex($direction: column);
+	border-spacing: 7px;
+	& thead {
```

Comment:
separate every block by an enter in each styles file. also add ".prettierrc" file and use an prettier extension for code formatting